irohad/torii/impl/command_service.cpp

```diff
@@ -39,44 +43,67 @@ namespace torii {
         storage_(storage),
         proposal_delay_(proposal_delay),
         start_tx_processing_duration_(1s),
-        cache_(std::make_shared<
-               iroha::cache::Cache<std::string,
-                                   iroha::protocol::ToriiResponse>>()) {
+        cache_(std::make_shared<CacheType>()) {
     // Notifier for all clients
     tx_processor_->transactionNotifier().subscribe(
         [this](
             std::shared_ptr<iroha::model::TransactionResponse> iroha_response) {
           // Find response in cache
-          auto res = cache_->findItem(iroha_response->tx_hash);
+          auto tx_hash = shared_model::crypto::Hash(iroha_response->tx_hash);
```

Comment:
Are you sure that `iroha_response->tx_hash` is in binary form?

Reply:
yes